viewer.py

```diff
@@ -606,12 +716,15 @@ def parseOntology(root, parent=-1):
     return ont
 
 
-def writeFile(data, file, **kwds):
```

Comment:
Yeah, I presume I added `**kwds` for a reason, although I don't remember what that was  :)